pywbem_mock/_dmtf_cim_schema.py

```diff
@@ -226,17 +226,38 @@ def schema_mof_dir(self):
         :term:`string`: Path name of the directory in which the DMTF CIM Schema
         MOF files are extracted from the downloaded zip file. This property can
         be used as the MOF compiler search path for compiling classes in the
-        DMTF CIM schema.
+        DMTF CIM schema. This directory should also contain the
+        cim_schema_.mof pragma file.
```

Comment:
Use a consistent term for that file. "DMTF CIM schema MOF pragma file" was used elsewhere.

Reply:
The correct name should be "schema_mof_pragma_file and the text should use the same words without the _.  Fixed some of them.  we may find some others.

DONE